src/parser/commands.h

```diff
@@ -1060,6 +1061,10 @@ class CVC5_EXPORT GetTimeoutCoreCommand : public Cmd
   cvc5::Solver* d_solver;
   /** The symbol manager we were invoked with */
   parser::SymManager* d_sm;
+  /** Did we provide soft constraints */
```

Comment:
```suggestion
  /** Did we provide assumptions */
```
I'd prefer assumptions instead of soft constraints as the latter is terminology from MaxSat.

Further, is this flag necessary? Couldn't this be checked via `!d_assumptions.empty()`?

Reply:
The flag was necessary because there could be the case where you ask for `get-timeout-core-assuming` with empty assumptions, which is different than asking `get-timeout-core`.

However, such a command is pointless since it just returns the empty set always. I will see if I can catch this / simplify.